openshift/4.0/templates/Logging/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.5"
+
+services:
+  logging:
+    restart: on-failure:1
+    container_name: pims-logging
+    build:
+      context: .
+    image: pims-logging:latest
+    environment:
+      SLEEP_TIME: 60
+      AZ_BLOB_URL: "https://pimsapp.blob.core.windows.net"
+      AZ_BLOB_CONTAINER: "pims"
+      AZ_SAS_TOKEN: "?sp=rcw&st=2021-07-04T06:11:43Z&se=2021-08-25T14:11:43Z&sv=2020-02-10&sr=c&sig=PRs%2FYfsUnAt2VJSO3XDzp8SUoLVqtNqZc7LDQdjeoXM%3D"
+      FRONTEND_APP_NAME: "pims-app"
+      API_NAME: "pims-api"
+      PROJECT_NAMESPACE: "3cd915-dev"
+      EXPORT_TIME: 360
+      OC_TOKEN: "sha256~UQsUaGV7mz9Sm-qwCaOrZx71H8JpoQ7ofQgRQfWKysA"
+      OC_SERVER: "https://api.silver.devops.gov.bc.ca:6443"
```

Comment:
I'd rather not see actual values here for **AZ_SAS_TOKEN** and **OC_TOKEN** - you should use env vars instead
Also I hope those are not real/valid tokens

Reply:
No the token are fake. and yes these are just for locally test. You will need a valid token

Reply:
@asanchezr made those chanages to use .env file as environment variable etc. Please, confirm